pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -93,96 +88,260 @@ func NewInspectCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
+
+		signedTimestamps, err := dumpSignedTimestamps(entity)
+		if err != nil {
+			return fmt.Errorf("failed to dump tsa: %w", err)
+		}
+		inspectedBundle.SignedTimestamps = signedTimestamps
+
+		inspectedBundles = append(inspectedBundles, inspectedBundle)
+	}
+
+	inspectionResult := BundleInspectResult{InspectedBundles: inspectedBundles}
+
+	// If the user provides the --format=json flag, print the results in JSON format
+	if opts.exporter != nil {
+		if err = opts.exporter.Write(opts.Logger.IO, inspectionResult); err != nil {
 			return fmt.Errorf("failed to write JSON output")
 		}
 		return nil
 	}
 
-	// otherwise, print results in a table
-	details, err := getDetailsAsSlice(opts.Tenant, results)
+	printInspectionSummary(opts.Logger, inspectionResult.InspectedBundles)
+
+	return nil
+}
+
+var heroHeader = `
+   _                       __ 
+  (_)__  ___ ___  ___ ____/ /_
+ / / _ \(_-</ _ \/ -_) __/ __/
+/_/_//_/___/ .__/\__/\__/\__/ 
+          /_/                 
+`
+
+func printInspectionSummary(logger *io.Handler, bundles []BundleInspection) {
+	logger.Println(heroHeader)
+
+	logger.Printf("Found %s:\n---\n", text.Pluralize(len(bundles), "attestation"))
+
+	bundleSummaries := make([][][]string, len(bundles))
+	for i, iB := range bundles {
+		bundleSummaries[i] = [][]string{
+			{"Authentic", formatAuthentic(iB.Authentic, iB.Certificate.CertificateIssuer)},
+			{"Source NWO", formatNwo(iB.Certificate.SourceRepositoryURI)},
+			{"PredicateType", iB.Statement.GetPredicateType()},
+			{"SubjectAlternativeName", iB.Certificate.SubjectAlternativeName},
+			{"RunInvocationURI", iB.Certificate.RunInvocationURI},
+			{"CertificateNotBefore", iB.Certificate.NotBefore.Format(time.RFC3339)},
+		}
+	}
+
+	// "SubjectAlternativeName" has 22 chars
+	maxNameLength := 22
+
+	scheme := logger.ColorScheme
+	for i, bundle := range bundleSummaries {
+		for _, pair := range bundle {
+			colName := pair[0]
+			dots := maxNameLength - len(colName)
+			logger.OutPrintf("%s:%s %s\n", scheme.Bold(colName), strings.Repeat(".", dots), pair[1])
+		}
+		if i < len(bundleSummaries)-1 {
+			logger.OutPrintln("---")
+		}
+	}
+}
+
+// formatNwo checks that longUrl is a valid URL and, if it is, extracts name/owner
+// from "https://githubinstance.com/name/owner/file/path@refs/heads/foo"
+func formatNwo(longUrl string) string {
+	parsedUrl, err := url.Parse(longUrl)
+
 	if err != nil {
-		return fmt.Errorf("failed to parse attestation details: %v", err)
+		return longUrl
 	}
 
-	headers := []string{"Repo Name", "Repo ID", "Org Name", "Org ID", "Workflow ID"}
-	t := tableprinter.New(opts.Logger.IO, tableprinter.WithHeader(headers...))
+	parts := strings.Split(parsedUrl.Path, "/")
+	if len(parts) > 2 {
+		return parts[1] + "/" + parts[2]
+	} else {
+		return parts[0]
+	}
+}
+
+func formatAuthentic(authentic bool, certIssuer string) string {
+	printIssuer := certIssuer
+
+	if strings.HasSuffix(certIssuer, "O=GitHub\\, Inc.") {
+		printIssuer = "(GH)"
```

Comment:
Maybe use gihub.com here isntead of `GH`?

Reply:
Regardless of what abbreviations we use for issuers, we might want to explain this within the command(s) `Long` usage?